XYZDrivers/src/main/webapp/index.html

```diff
@@ -1,10 +1,44 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Start Page</title>
+        <title>Home Page</title>
```

Comment:
When I run the app as a whole I'm still being taken to the old "Hello World!" page. It doesn't seem to be defaulting to http://localhost:8080/XYZDrivers/index.html. Instead it defaults to http://localhost:8080/XYZDrivers/. Does something in the web.xml need to be changed? Pretty sure as standard it goes to index.html but I don't know if something has been modifed to change that.